CenterForOpenScience/ember-osf-web
# [ENG-670] improved collections submit acceptance test

- Ticket: [ENG-670]
- Feature flag: n/a

## Purpose

This PR adds test selectors and improves the collections submit acceptance test to perform actual assertions (previously just Percy snapshots).

## Summary of Changes

### Changed
- Components
    - `project-contributors`
        - added `onAddContributor` hook
- Engines
    - `collections`
        - Tests
            - added/improved test selectors to templates related to submit
            - improved submit acceptance tests to perform assertions in addition to taking snapshots
- Tests
    - added `ember-basic-dropdown-wormhole` div to test index.html

### Fixed
- Engines
    - `collections`
        - `submit`
            - reload bibliographicContributors when adding a contributor

## Side Effects

None expected.

## QA Notes

This should be tested as part of the full regression. The only functional change in this PR is to reload bibliographic contributors after adding a new contributor so that the contributor list displayed when the project contributors panel is collapsed is updated.


[ENG-670]: https://openscience.atlassian.net/browse/ENG-670
